docs: add upgrade guide (#857)
Refinitiv/refinitiv-ui: documents/src/pages/start/upgrade-guide.md

<!--
type: page

> title: Element Framework 7 Upgrade Guide

Reviewer: ```suggestion
title: Upgrade Guide
```

location: ./start/upgrade-guide
layout: default
-->

> # Element Framework 7 Upgrade Guide

Reviewer: ```suggestion
# Upgrade Guide
```

> This guide describes upgrading steps from Element Framework 6(EFv6) to Element Framework 7(EFv7). EFv7 has very few breaking changes from EFv6. Any projects using EFv6 without deprecation warnings or not using `ef-chart` and `ef-interactive-chart` should not require any changes to upgrade to EFv7.

Reviewer: ```suggestion
This guide describes upgrading steps for Element Framework from EFv6 to EFv7. EFv7 has very few breaking changes from EFv6. Any projects using EFv6 without deprecation warnings, not using `ef-chart` or `ef-interactive-chart` should not require any changes to upgrade to EFv7.
```
How about this one?

```

## Migrating Interactive Chart (ef-interactive-chart)
Chart library that used by `ef-interactive-chart`, [lightweight chart](https://www.tradingview.com/lightweight-charts/), has been upgraded from v3 to v4. As it's a major upgrade version, there are changes in configuration and some APIs of lightweight chart are deprecated. You can follow migration guides provided by lightweight chart team in [v4 migration guide](https://tradingview.github.io/lightweight-charts/docs/next/migrations/from-v3-to-v4).

Example:
* `scaleMargins` option has been removed from series options
* `backgroundColor` from layout options has been removed

__ef-interactive-chart v6 configuration__
```javascript
const config = {
  series: [{
    type : 'volume',
      layout: {
        backgroundColor: 'red'
      },
    seriesOptions: {
      scaleMargins: {
        top: 0.8,
          bottom: 0
      }
    }
 }]
};
```

__ef-interactive-chart v7 configuration__
```javascript
const config = {
  series: [{
    type : 'volume',
    layout: {
      background: {
        color: 'red'
      }
    }
  }]
};

chart.priceScale('').applyOptions({
  scaleMargins: {
    top: 0.8,
    bottom: 0
  }
});
```

> ## Guideline For EF Extension Team (EFX)

Reviewer: Does EFX Team also need guideline for interactive-chart ?
